RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -856,6 +858,7 @@ string freeRunCombatString(monster enemy, location loc, boolean inCombat)
 	{
 		// TODO add fam weight buffing
 		int banderRunsLeft = floor((familiar_weight($familiar[Frumious Bandersnatch]) + weight_adjustment()) / 5) - get_property("_banderRunaways").to_int();
+		if(in_wereprof() && !is_werewolf()) return "";
```

Comment:
can this be removed? Already checked on line 825

Reply:
No it can't. These lines prevent bandersnatch and stomping boots from being used as a professor, because these are skills and the professor can't use skills. Line 825 makes it so that we can't freerun as a Werewolf because the Werewolf doesn't run from fights.